cypress/e2e/catalogue/catalogueCategory.cy.ts

```diff
@@ -26,6 +41,33 @@ describe('Catalogue Category', () => {
     cy.findByText('actuators').should('not.exist');
   });
 
+  it('should be able to change page', () => {
+    cy.editEndpointResponse({
+      url: '/v1/catalogue-categories/',
+      data: createMockData(),
+      statusCode: 200,
+    });
+
+    cy.findByText('Test 1').should('exist');
+    cy.findByRole('button', { name: 'Go to page 2' }).click();
+
+    cy.findByText('Test 31').should('exist');
+  });
+
+  it('should be able to change max results', () => {
+    cy.editEndpointResponse({
+      url: '/v1/catalogue-categories/',
+      data: createMockData(),
+      statusCode: 200,
+    });
+
+    cy.findByText('Test 1').should('exist');
```

Comment:
```suggestion
    cy.findByText('Test 1').should('exist');
    cy.findByText('Test 45').should('not.exist');
```
Just to ensure it wasn't there before